mne/io/edf/tests/test_edf.py

```diff
@@ -52,8 +53,8 @@ def test_concat():
 
 def test_bdf_data():
     """Test reading raw bdf files"""
-    raw_py = read_raw_edf(bdf_path, montage=montage_path, eog=eog,
-                          misc=misc, preload=True)
+    raw_py = _test_raw_object(read_raw_edf, True, input_fname=bdf_path,
```

Comment:
Okay. Keep in mind there are actually there `preload` modes, not two, though: False (on demand), True (in memory) and str (memmapping), so you probably want to add a third. This is one of those that is probably most completely tested in `fiff/tests/test_raw.py`.